anta/tests/routing/bgp.py

```diff
@@ -218,8 +220,12 @@ def render(self, template: AntaTemplate) -> list[AntaCommand]:
         """Render the template for each BGP address family in the input list."""
```

Comment:
maybe this is a docstring that deserves more info given the "dark magic" in the rendering function